extensions/amp-analytics/amp-analytics.md

```diff
@@ -559,6 +561,24 @@ The selector properties are:
     of the `amp-analytics` tag that satisfies the given selector. The default
     value is `scope`.
 
+##### Selector Arguments <a name="available-triggers"></a>
```

Comment:
`<a name="available-triggers"` I think this is something special for amp.dev. Let's remove it. 

Also, I'm not a fan of `Arguments` since we use arguments already for some variable substitutions. WDYT about `Selector Values`?

Reply:
Okay that makes sense. Just made the change.